charts/nublado/templates/prepuller/role.template.yml

```diff
@@ -5,5 +5,5 @@ metadata:
   name: {{ template "nublado.fullname" . }}-prepuller
 rules:
 - apiGroups: [""]
-  resources: ["pods"]
+  resources: ["pods", "secrets"]
```

Comment:
I think this is fine but on a different PR Russ and I were talking about trying to make this a bit tighter for v2 (and I copied over what v1's role was, not sure what we need, but we can figure that out).

Reply:
Yeah, we can probably tighten it, but the prepuller may not even exist in anything like the current form.